packages/specs/schema/src/components/open-spec/operationInQueryMapper.ts

```
@@ -33,12 +34,20 @@ function buildExamples(property: string, examples?: Record<string, OS3Example>)
+    jsonParameter,
+    ...options
+  }: JsonSchemaOptions & {
+    jsonParameter: JsonParameter;
+  }
+) {
+  const name = createRefName(jsonParameter.schema().itemSchema().getName(), options);
   const schema = options.components?.schemas?.[name];
 
   if (schema && !options.oldSchemas?.[name]) {
-    delete options.components!.schemas![jsonParameter.$schema.getName()];
+    delete options.components!.schemas![jsonParameter.schema().itemSchema().getName()];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)